prog/size_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2016 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package prog
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestAssignSizeRandom(t *testing.T) {
+	rs, iters := initTest(t)
+	for i := 0; i < iters; i++ {
+		p := Generate(rs, 10, nil)
+		for _, call := range p.Calls {
+			assignSizesCall(call)
```

Comment:
The generated program already has all sizes assigned, so this should not change the program. It may be useful to check this:

```
data0 := p.Serialize()
... re-assign sizes ...
if data1 := p.Serialize(); !bytes.Equal(data0, data1) {
  t.Fatalf(...)
}
```